app-misc/vcontrold/vcontrold-0.98.5.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Your calendar is running a bit late.

Reply:
Updated with 02a680548562a18c0f710438f2ac00f2e8df816d